core/src/main/scala/org/apache/spark/sql/store/StoreUtils.scala

```diff
@@ -414,15 +414,16 @@ object StoreUtils {
     parameters.remove(PARTITIONER).foreach(v =>
       sb.append(GEM_PARTITIONER).append('\'').append(v).append("' "))
 
-    val defaultEviction = s"$GEM_HEAPPERCENT $GEM_OVERFLOW"
+    val overflow = parameters.get(OVERFLOW).map((_.toBoolean)).getOrElse(true)
```

Comment:
This can lead to bad error btw if the user makes some spelling mistake in writing "false" or "true".
Blindly doing _.toBoolean will give error.

Reply:
It throws below message, which user can use to find out the issue. This should be okay, right?
`
snappy> create table coltab (id int) using column options (overflow 'fals');
ERROR 38000: (SQLState=38000 Severity=20000) (Server=localhost/127.0.0.1[1528] Thread=ThriftProcessor-0) The exception 'com.pivotal.gemfirexd.internal.engine.jdbc.GemFireXDRuntimeException: myID: 127.0.0.1(26557)<v1>:5443, caused by java.lang.IllegalArgumentException: For input string: "fals"' was thrown while evaluating an expression.
`